cmd/webhook/app/webhook.go

```diff
@@ -158,6 +160,8 @@ func Run(ctx context.Context, opts *options.Options) error {
 	hookServer.Register("/mutate-multiclusterservice", &webhook.Admission{Handler: &multiclusterservice.MutatingAdmission{Decoder: decoder}})
 	hookServer.Register("/mutate-federatedhpa", &webhook.Admission{Handler: &federatedhpa.MutatingAdmission{Decoder: decoder}})
 	hookServer.Register("/validate-resourcedeletionprotection", &webhook.Admission{Handler: &resourcedeletionprotection.ValidatingAdmission{Decoder: decoder}})
+	hookServer.Register("/mutate-resourcebinding", &webhook.Admission{Handler: &resourcebinding.MutatingAdmission{Decoder: decoder}})
```

Comment:
I'm not sure if adding this is reasonable, but it seems it can also be handled in the controller without adding it.

Reply:
It's a little bit complex and difficult to manage the code in controller, I prefer to add this in weebhook, not in controller

Reply:
/cc @RainbowMango

Reply:
@chaunceyjiang What's your concern?
I can tell a challenge for the user to update the `MutatingWebhookConfiguration` during upgrading.

Reply:
Actually I waiting for @chaunceyjiang about this :)